testsuites/syncgateway/functional/tests/test_attachments.py

```diff
@@ -81,7 +82,10 @@ def test_attachment_revpos_when_ancestor_unavailable(params_from_base_test_setup
     # Clear cached rev doc bodys from server and cycle sync_gateway
     sg_util.stop_sync_gateways(cluster_config=cluster_config, url=sg_url)
 
-    cb_server.delete_couchbase_server_cached_rev_bodies(bucket=bucket)
+    ipv6 = False
+    if is_ipv6:
```

Comment:
is_ipv6 function requires cluster_config as an argument, right? I wonder what it will return if calling with passing cluster_config?

Reply:
yup, it need it. Made changes.